### ISSUE TITLE
otffccdump 0.6.0 Segmentation fault while JSON last brace

### ISSUE THREAD

**Issue by cwchien** [caryll/otfcc#41]
Source Han Sans 1.004
for example: [SourceHanSansCN-Bold.otf](https://github.com/adobe-fonts/source-han-sans/blob/release/SubsetOTF/CN/SourceHanSansCN-Bold.otf)
``` console
$ otfccdump -v
This is otfccdump, version 0.6.0.
$ otfccdump SourceHanSansCN-Bold.otf
......
            "latn": {
                "defaultBaseline": "romn",
                "baselines": {
                    "icfb": 35,
                    "icft": 965,
                    "ideo": 0,
                    "romn": 120
                }
            }
        }
    }
Segmentation fault
```
compared with otfccdump 0.5.2 for the same file:
``` console
$ otfccdump -v
This is otfccdump, version 0.5.2.
$ otfccdump SourceHanSansCN-Bold.otf
......
            "latn": {
                "defaultBaseline": "romn",
                "baselines": {
                    "icfb": 35,
                    "icft": 965,
                    "ideo": 0,
                    "romn": 120
                }
            }
        }
    }
}
```

OS: Ubuntu 16.10 64bit
GCC: 6.2.0 20161005
Premake : 5.0.0-alpha10
GMake: 4.1
otfcc built with:
``` console
$premake5 gmake
$cd build/gmake/
$make config=release_x64
```
otfcc-0.6.0 binary [Google Drive](https://drive.google.com/drive/folders/0B26DiRHGijjEUXNORWhnMUdsMlU?usp=sharing)

**Comment by be5invis** [comment]
Fixed in 0.6.1.

**Comment by cwchien** [comment]
Confirmed. Thanks. :)
